extensions/amp-sona/0.1/test/validator-amp-sona.out

```diff
@@ -0,0 +1,43 @@
+PASS
+|  <!--
+|    Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Can you update this value to 2020? The value should match the year the new file is committed.

Reply:
👍